### ISSUE TITLE
 Add credits

### ISSUE THREAD

**Issue by theheroGAC** [theheroGAC/Autoplugin#136]
It would be nice to see all the developers who collaborated on this project and the Autoplugin developers / translators on a page

**Comment by bamhm182** [comment]
What sections and orders do we want? I was thinking Creator, Developers, Translators and having translators listed like:
name (language)
name (language)

As far as the order, I was thinking alphabetical because it's the easiest, but sorting by amount contributed would be a better ordering.

**Comment by theheroGAC** [comment]
Exactly. Let's start with the programming language, the credits to the OneLua Team and then the collaborator developers etc. Obviously like the ending of a film
